packages/workflows/src/sync.workflows.ts

```diff
@@ -132,34 +132,51 @@ const workflows: IDeploymentWorkflows = {
         function: async ({ tasks, config, options }) => {
           // HACK - ensure drive id provided as array (can be removed once deprecation removed)
           const { assets_folders } = migrateLegacyGdriveConfig(config.google_drive);
-          /** Return output of paths to downloaded assets */
-          let outputs: string[] = [];
+          /** Return output of paths to downloaded assets and folder metadata */
+          let outputs: Array<{ path: string; folderConfig: (typeof assets_folders)[0] }> = [];
           // If skipping download still need to return download folder for next step
           if (options.skipDownload) {
-            outputs = assets_folders.map(({ id, name }) =>
-              tasks.gdrive.getOutputFolder("assets", id, name)
-            );
+            outputs = assets_folders.map((folderConfig) => ({
+              path: tasks.gdrive.getOutputFolder("assets", folderConfig.id, folderConfig.name),
+              folderConfig,
+            }));
           } else {
             const { assets_filter_function } = config.google_drive;
-            for (const { id: folderId, name: folderName } of assets_folders) {
+            for (const folderConfig of assets_folders) {
               const output = await tasks.gdrive.download({
                 type: "assets",
-                folderId,
-                folderName,
+                folderId: folderConfig.id,
+                folderName: folderConfig.name,
                 filterFn: assets_filter_function,
               });
-              outputs.push(output);
+              outputs.push({ path: output, folderConfig });
             }
           }
           return outputs;
         },
       },
       {
         name: "assets_post_process",
-        function: async ({ tasks, workflow }) =>
-          tasks.appData.postProcessAssets({
-            sourceAssetsFolders: workflow.assets_dl.output,
-          }),
+        function: async ({ tasks, workflow }) => {
+          // Build folder metadata map from download results
+          const folderMetadata = new Map<string, { remote?: boolean; folderName?: string }>();
+          const sourceAssetsFolders: string[] = [];
+
+          for (const { path, folderConfig } of workflow.assets_dl.output) {
+            sourceAssetsFolders.push(path);
+            if (folderConfig.remote) {
+              folderMetadata.set(path, {
+                remote: true,
+                folderName: folderConfig.name,
+              });
+            }
+          }
+
+          return tasks.appData.postProcessAssets({
+            sourceAssetsFolders,
+            folderMetadata,
+          });
+        },
```

Comment:
nit(blocking): I would still expect to see one additional processing step, which runs conditionally when the deployment has enable remote asset deployment config. Something like `sync_remote_assets` which handles the final step of pushing new assets to the server and deleting outdated (TBC how best to manage given legacy app users may still want access to the packs not required in latest release...)

Short term this is likely just copying from one local folder to another for visibility (if not already done so in previous processing step)